reporter/src/test/kotlin/reporters/NoticeReporterTest.kt

```diff
@@ -40,34 +40,34 @@ class NoticeReporterTest : WordSpec() {
         "Notices reporter" should {
             "generate the correct license notes" {
                 val expectedText = File("src/test/assets/NPM-is-windows-1.0.2-expected-NOTICE").readText()
-                val ortResult = readOrtResult("src/test/assets/NPM-is-windows-1.0.2-scan-result.json")
+                val scanResult = readOrtResult("src/test/assets/NPM-is-windows-1.0.2-scan-result.json")
```

Comment:
The name `ortResult` better reflects what the object contains. A `scanResult` is only one component of an `ortResult`. Also, the notice reporter does not only use the scan result, but also the analyzer result to get declared licenses and the repository configuration to get excludes.